tests/Unit/Domain/Test_Block.cpp

```diff
@@ -0,0 +1,64 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include <catch.hpp>
+
+#include "Domain/Block.cpp"
+#include "Domain/EmbeddingMaps/Identity.hpp"
+#include "tests/Unit/TestHelpers.hpp"
+
+template <size_t Dim>
+void test_identity() {
+  const EmbeddingMaps::Identity<Dim> identity_map;
+  const Block<Dim> block(identity_map, 7, {});
+
+  // Test external boundaries:
+  CHECK((block.external_boundaries().size()) == 2 * Dim);
+
+  // Test neighbors:
+  CHECK((block.neighbors().size()) == 0);
+
+  // Test id:
+  CHECK((block.id()) == 7);
+
+  // Test that the block's embedding_map is Identity:
+  const auto& map = block.embedding_map();
+  const Point<Dim, Frame::Logical> xi(1.0);
+  const Point<Dim, Frame::Grid> x(1.0);
+  CHECK(map(xi) == x);
+  CHECK(map.inverse(x) == xi);
```

Comment:
Blocks have unique_ptrs to EmbeddingMap - I'm unsure if we want to compare unique ptrs? (Since we don't have comparison for EmbeddingMaps themselves)